src/pages/guides/using/custom_events.md

```diff
@@ -0,0 +1,31 @@
+# Custom I/O Events Overview
```

Comment:
need to add this to gatsby-config.js

Reply:
Good catch. Will do that!

Reply:
Done. Btw I just realized that adding the pages is only required for side and global navigation. They will get rendered if they are linked from somewhere else.

Reply:
yes, i felt we do need it to be part of the side menu.